src/app/project/board/dataset-metadata.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { IId, Organization, Person } from '@dasch-swiss/dsp-js';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DatasetMetadataService {
+
+  constructor() { }
+
+  /**
+  * used in project metadata page
+  * determine if the object is of type Person or Organization or Iid
+  * @param obj Person | Organization | IId
+  */
+  getContactType(obj: Person | Organization | IId): string {
+    if (obj instanceof Person) {
+      return 'person';
+    }
+    if (obj instanceof Organization) {
+      return 'organization';
+    }
+    return undefined;
+  }
```

Comment:
Use `if..else if` block here, [this answer](https://github.com/dasch-swiss/dsp-app/pull/400#discussion_r585398051).

Reply:
Thanks. Changes are done in [d8b4993](https://github.com/dasch-swiss/dsp-app/pull/400/commits/d8b4993b6648e3fe469ecdf0016843e8d0b8d78c).